packages/variable-editor/src/VariableEditor.tsx

```diff
@@ -99,11 +94,11 @@ function VariableEditor(props: EditorProps) {
       masterTitle={title}
       masterContent={
         <VariablesMaster
+          context={context}
```

Comment:
instead of move this context prop down, we could also hold this inside of a react context.
But totally fine for now :)

Reply:
Yes, I have seen this in inscription view. I will let it like this for now. We can improve if we add more features.